### ISSUE TITLE
app-crypt/haiti-hash-1.2.2 fails tests: No Rakefile found 

### ISSUE THREAD

**Issue by asarubbo** [pentoo/pentoo-overlay#760]
https://blogs.gentoo.org/ago/2020/07/04/gentoo-tinderbox/

Issue: app-crypt/haiti-hash-1.2.2 fails tests.
Discovered on: amd64 (internal ref: pentoo_tinderbox)

At the time of writing, github-cli does not support attachments. I'm using gists but they can be deleted at a certain point. If you want to save them, please create attachments on your own.

build log: https://gist.github.com/6fbb54cfe731a4f3091320394b4ea48d

**Comment by blshkv** [comment]
Yes, yes, we know. Our ebuilds are not perfect, that's why it is still not in Gentoo. Please create a pull request with a fix you care

**Comment by alsotoes** [comment]
This version is no longer supported, currents are:

[haiti-hash-1.2.3.ebuild]
[haiti-hash-1.3.0.ebuild]

A new test should test those, IMHO I believe this issue can be closed, cc/ @ZeroChaos- @blshkv

**Comment by ZeroChaos-** [comment]
That's not how this work.  This is an ebuild issue, updating the upstream version won't fix it.  Please stop telling me to close bugs when no work has been done, I know how to close bugs.
